### DISCUSSION TITLE
An error occurred.

### DISCUSSION THREAD

**Discussion by IMHFX** [bs-community/blessing-skin-server#452] [Q&A]
服务器系统：CentOs8
地址：192.168.1.5/setup（局域网）
浏览器：edge
服务器没有报错，也没有日志
<img width="960" alt="屏幕截图 2022-07-26 183208" src="https://user-images.githubusercontent.com/110028134/180985954-3238f91c-fa0d-49f5-80ed-8f004baef20d.png">
[nginx.conf.txt](https://github.com/bs-community/blessing-skin-manual/files/9188998/nginx.conf.txt)

**Comment by g-plane** [comment]
请仔细、认真按手册的指引进行搭建。PHP 环境搭建相关的资料请自行在其它地方搜索。

**Comment by IMHFX** [comment]
😅好像什么都说了，又好像什么都没说

**Reply by Big-Cake-jpg** [reply]
照你这意思是开发者得手把手给你搭建喽

**Comment by gxres042** [comment]
> 😅好像什么都说了，又好像什么都没说

手册只是拿来记载有用的信息的，并不是拿来记载教你从零开始搭建 Blessing Skin Server 的。请记住这一点。
